packages/boba/account-abstraction/README.md

```diff
@@ -0,0 +1,7 @@
+Implementation of contracts for [ERC 4337](https://github.com/ethereum/EIPs/blob/3fd65b1a782912bfc18cb975c62c55f733c7c96e/EIPS/eip-4337.md) account abstraction via alternative mempool.
+
+# Resources
+
+[Vitalik's post on account abstraction without Ethereum protocol changes](https://medium.com/infinitism/erc-4337-account-abstraction-without-ethereum-protocol-changes-d75c9d94dc4a)
+
+[Discord server](http://discord.gg/fbDyENb6Y9)
```

Comment:
I see this links to Infinitism's discord? Do we want to leave that?

Reply:
@souradeep-das Any suggestions here?